fe/src/main/java/org/apache/doris/load/routineload/RLTaskTxnCommitAttachment.java

```diff
@@ -29,20 +29,20 @@
 // "numOfTotalData": "", "taskId": "", "jobId": ""}
 public class RLTaskTxnCommitAttachment extends TxnCommitAttachment {
 
-    public enum RoutineLoadType {
+    public enum LoadSourceType {
```

Comment:
There is a `LoadDataSourceType`enum in package `routineload`.  Using 'LoadDataSourceType' is better